src/style.css

```diff
@@ -1 +1,50 @@
 /* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars__star {
+  background-image: url('images/star.svg');
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
Use https://developer.mozilla.org/ru/docs/Web/CSS/background
To combine `background-repeat`, `background-position` and `background-image` over just one `background`